common/src/app/common/types/shape/interactions.cljc

```diff
@@ -262,6 +263,13 @@
     (assoc :overlay-pos-type :center
            :overlay-position (gpt/point 0 0))))
 
+(defn set-position-relative-to
+  [interaction position-relative-to]
+  (us/verify ::interaction interaction)
+  (us/verify ::position-relative-to position-relative-to)
+  (assert (has-destination interaction))
```

Comment:
I understand that the new `:position-relative-to` parameter is related only to overlays, isn't it?

If so, this should be `(has-overlay-opts interaction)`

Reply:
Mmm... I need it there to initialize the value to "self" on new interactions. Maybe I can initialize the value in another place, I need to check...

Reply:
Maybe call to `set-position-relative-to` when you change the action type, and only if `(has-overlay-opts` is true.

Reply:
Ok, I found a way :)